experiments/defaults.py

```diff
@@ -589,11 +589,7 @@ def _prepare_data_config(
         validation_sets = {}
 
     if isinstance(tokenized, InputName | ExecutorStep):
-        pretraining_data = lm_data_config(
-            training_set=tokenized,
-            validation_sets=validation_sets,
-            permutation_type="linear",
-        )
+        pretraining_data = lm_data_config(training_set=tokenized, validation_sets=validation_sets)
```

Comment:
my intent was to expose permutation_type as a parameter, fix all archival usages to linear then set the default to feistel, but if we're willing to just let that go lgtm. @percyliang wdyt